cloud_kafka/librdkafka/avro_producer.cc

```diff
@@ -0,0 +1,132 @@
+#include <stdio.h>
```

Comment:
```suggestion
#include <cstdio>
```

Reply:
`cstdio` only provides the same functions and structs in the `std` namespace (e.g. `std::printf` is available in this header).